IO/src/main/java/io/deephaven/io/log/LogEntry.java

```diff
@@ -125,7 +125,8 @@ default LogEntry appendDouble(final double doubleValue, final int decimalPlaces,
                 decimalPlaces - maxTrailingZeroesToDiscard);
     }
 
-    private static LogEntry appendDoubleDecimalPlacesImpl(
+
+    static LogEntry appendDoubleDecimalPlacesImpl(
```

Comment:
This should stay `private`: I don't see any non-private uses, and with the name it doesn't seem like a an appropriate public method.

Reply:
You can't have private methods in an interface in Java 8, that support only happens in 9 and above

Reply:
Maybe make a `@InternalUseOnly Helper` class that's `public`, and make the method a `private` static member of that class?